scripts/webpCompression.js

```diff
@@ -0,0 +1,138 @@
+const { exec } = require("child_process");
+const fs = require("fs");
+const fsWalk = require("walk");
+const minimist = require("minimist");
+const path = require("path");
+
+function parseBool(str) {
+	if (typeof str === "string") {
+		str = str.toLowerCase();
+		if (str === "true" || str === "false") {
+			return str === "true";
+		}
+	}
+	return Boolean(str);
+}
+
+function parsePath(key, name, def) {
+	if (key in argv) {
+		const temp = argv[key];
+		if (typeof temp !== "string") {
+			console.log(`\x1b[31m${name} must be a string.\x1b[0m`);
+			process.exit(9);
+		}
+		return temp;
+	}
+	return def;
+}
+
+function parsePresets(set) {
+	const presets = [
+		"cwebp -m 6 -q 100 -mt -v",
```

Comment:
"cwebp" will this command run on any machine ? Haven't seen a mention in the package.json

Reply:
https://github.com/Coldknife2/JourneySaveParserHtml/pull/18/commits/f542d8888b5e216c327eb3eb076b82e036172706 resolves that by checking if the needed binaries are available.